plugins/source/stripe/resources/services/products/products_test.go

```diff
@@ -0,0 +1,18 @@
+package products_test
+
+import (
+	"testing"
+
+	"github.com/cloudquery/cloudquery/plugins/source/stripe/client"
+	"github.com/cloudquery/cloudquery/plugins/source/stripe/resources/services/products"
+)
+
+func TestProductsProduct(t *testing.T) {
+	tbl := products.Products()
+	for i, c := range tbl.Columns {
+		if c.Name == "attributes" || c.Name == "deactivate_on" {
+			tbl.Columns[i].IgnoreInTests = true
```

Comment:
We need to do this because the mock server doesn't return the column even though the fixture is populated correctly.